composer.json

```diff
@@ -12,7 +12,8 @@
         }
     ],
     "require": {
-        "php": ">=5.6"
+        "php": ">=5.6",
+        "ext-mbstring": "*"
```

Comment:
My only concern about it is that there are 144 rules and just because 8 rules we insert this dependency.